tools/driver/p4c.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+from p4c.main import main
+
+if __name__ == '__main__':
+    main()
```

Comment:
quick question: since the rest of the repo is using autotools, why not use autotools to install the driver as well?